proofs/verification/rustverifier.go

```diff
@@ -0,0 +1,45 @@
+package verification
+
+import "github.com/filecoin-project/go-filecoin/proofs/libsectorbuilder"
+
+// RustVerifier provides proof-verification methods.
+type RustVerifier struct{}
```

Comment:
Is this struct just for namespacing within the package?  Trying to understand the choice to not make these free functions.

Reply:
Totally legitimate question. There exist other structs which satisfy the Verifier interface (for DI purposes), and some functions accept a Verifier. We could modify the functions which accept a Verifier to instead accept two functions of the appropriate types, but the interface and struct approach is consistent with the rest o’ the codebase.